src/smbprotocol/structure.py

```diff
@@ -161,6 +160,10 @@ def __str__(self):
     def __len__(self):
         return self._get_packed_size()
 
+    @property
+    def endian_prefix(self):
```

Comment:
Prefer to keep this as `_endian_prefix` just to make sure it's more of an internal property.

Reply:
Okay